changelogs/unreleased/4200-module-install-change-requirements.yml

```
@@ -0,0 +1,4 @@
+description: Dummy change entry for a change made in master/iso5 that was finally not ported to iso4
+issue-nr: 3986
+change-type: patch
+destination-branches: [master, iso5, iso4]
```

```suggestion
destination-branches: [iso4]
```
I hadn't explained this: when cherry-picking (or creating a dummy), you should trim the set of destination branches down to the set of branches the cherry-pick is meant for. The nightly invariants check allows this.